service-front/web/main.scss

```
@@ -12,3 +12,28 @@ $govuk-assets-path: "../assets/";
 .moj-banner--date {
   max-width: 150px;
 }
+
+.app-inset-text-blue {
+  font-family: "GDS Transport", arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  font-size: 1rem;
+  line-height: 1.25;
+  color: #0b0c0c;
+  padding: 15px;
+  margin-top: 20px;
+  margin-bottom: 20px;
```

This can still be switched to variables. Particularly the typography parts
```suggestion
  @include govuk-font($size: 16);
  color: $govuk-text-colour;
  padding: govuk-spacing(3);
  margin-top: govuk-spacing(4);
  margin-bottom: govuk-spacing(4);
```

---

Sorry about this - I used the accept change button on github on this ones, seems not to have worked